keras/regularizers.py

```diff
@@ -16,6 +17,45 @@ def get_config(self):
         return {'name': self.__class__.__name__}
 
 
+class EigenvalueRegularizer(Regularizer):
+    """This class implements the Eigenvalue Decay regularizer.
+    
+    Args:
```

Comment:
The dosctring format should match other docstrings found throughout the codebase.